sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AadAuthenticationFilterAutoConfiguration.java

```diff
@@ -60,7 +60,7 @@ public class AadAuthenticationFilterAutoConfiguration {
      */
     public AadAuthenticationFilterAutoConfiguration(
             AadAuthenticationProperties properties,
-            @Qualifier(AAD_OAUTH_2_RESOURCE_SERVER_REST_OPERATION_BEAN_NAME) RestOperations restOperations) {
+            @Qualifier(AAD_OAUTH2_RESOURCE_SERVER_REST_OPERATION_BEAN_NAME) RestOperations restOperations) {
```

Comment:
Should we call these `REST_OPERATIONS` to match the class name?

Reply:
Updated.